src/main/java/core/basesyntax/Excavator.java

```diff
@@ -0,0 +1,13 @@
+package core.basesyntax;
+
+public class Excavator extends Machine {
+    @Override
+    public void doWork() {
+        System.out.println("The excavator has started work");

```

Comment:
The message printed should be more informative about the type of machine. Consider specifying that it is an Excavator that has started working, to make it clear which machine is being referred to.